app/views/session.py

```diff
@@ -69,8 +70,8 @@ def login():
     cookie_session['theme'] = g.schema.json['theme']
     cookie_session['survey_title'] = g.schema.json['title']
 
-    if 'account_service_url' in claims and claims.get('account_service_url'):
-        cookie_session[ACCOUNT_URL] = claims.get('account_service_url')
+    if 'account_service_url' in claims:
```

Comment:
It can never be empty or None in claims?

Reply:
Or, to clarify, previously we didn't set the url if it was None in claims. Now we will. Just want to make sure that's expected.

Reply:
Thats a good point...I'll put that back to what it was.  There's no adverse effects from this, but there's no point in adding keys with empty values in the session if we can avoid it.